biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.css

```diff
@@ -1,68 +1,71 @@
-#biogridVisual {
-  width: 28.125rem;
-}
-
 body {
-  background-color: #3b6d6d;
+  background-color: #0b5a5a;
 }
 
 br {
   clear: both;
 }
 
-.buildings {
+.buildings, .endDatePicker, .largeBatteryCells, 
+.smallBatteryCells, .solarPanels, .startDatePicker, 
+.townHeight, .townWidth {
```

Comment:
instead of listing each of these classes, can you pick one class name (e.g. `gridItem`) and in the HTML, make sure each of these items has  `gridItem` class

Reply:
Wow..never realized this. Thank you for pointing this out!